src/main/sql/add_user_roles.sql

```diff
@@ -0,0 +1,58 @@
+-- This script creates roles which will be be available for assigning to users later.
+--/
+CREATE OR REPLACE SCRIPT row_level_security_test_schema.add_user_role(role_name) returns table AS
+        log = {}
+        table_name = 'EXA_ROLES_MAPPING'
+
+        function add_role_into_new_table()
+                log[#log+1] = {"Table "..table_name.." not found. Creating a new table"}
+                pquery([[CREATE TABLE ::table_name (EXA_ROLE VARCHAR(120),
```

Comment:
All identifiers in Exasol can be up to 128 characters long. We should mimic that.
```suggestion
                pquery([[CREATE TABLE ::table_name (EXA_ROLE VARCHAR(128),
```

Reply:
outdated